Lib/test/test_json/test_tool.py

```diff
@@ -92,6 +95,22 @@ def test_infile_outfile(self):
         self.assertEqual(out, b'')
         self.assertEqual(err, b'')
 
+    def test_infile_same_outfile(self):
+        infile = self._create_infile()
+        rc, out, err = assert_python_ok('-m', 'json.tool', infile, infile)
+        self.assertEqual(rc, 0)
```

Comment:
you don't need to check the value of rc, `assert_python_ok` already does it for you.

Reply:
I already did this mistake in the past, but I think we could create an other PR where we remove these duplicated tests `self.assertEqual(rc, 0)`

Reply:
Another PR would be great.